Opening repositories:
C:\Users\phili\Desktop\Programming\Capstone\VapeInventory\VapeInventory
Opening repositories:
C:\Users\phili\Desktop\Programming\Capstone\repo
Opening repositories:
C:\Users\phili\Desktop\Programming\Capstone\VapeInventory\VapeInventory
Commit c40e3a74 created locally in repository C:\Users\phili\Desktop\Programming\Capstone\VapeInventory\VapeInventory
Opening repositories:
C:\Users\phili\Desktop\Programming\Capstone\VapeInventory\VapeInventory
The calling thread cannot access this object because a different thread owns it.
Pushing master
Error: failed to push some refs to 'https://github.com/redthre3/CapstoneVapeInventory.git'
Error: hint: Updates were rejected because the remote contains work that you do
hint: not have locally. This is usually caused by another repository pushing
hint: to the same ref. You may want to first integrate the remote changes
hint: (e.g., 'git pull ...') before pushing again.
hint: See the 'Note about fast-forwards' in 'git push --help' for details.

Error encountered while pushing branch to the remote repository: rejected Updates were rejected because the remote contains work that you do not have locally. This is usually caused by another repository pushing to the same ref. You may want to first integrate the remote changes before pushing again.
